### PR TITLE
Added Docker setup

### PR DESCRIPTION
**Pull request by vidarl**
Simple  config for developing inside a docker container

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:latest
+
+ARG USER_ID
+ARG GROUP_ID
+
+RUN apt-get update -q -y && apt-get install -y php php-xml php-zip unzip openssh-client composer
```
[the remark this answers is not in the document]
Comment (vidarl) [reply Dockerfile]: Honestly, I don't remember why I added php-xml, php-zip and unzip... I have removed them again.

I guess it is a matter of preferences if you want to install something via distro packages or not. I have complied with your request though

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:latest
```
[the remark this answers is not in the document]
Comment (vidarl) [reply Dockerfile]: Done

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+ARG USER_ID
+ARG GROUP_ID
+
+RUN apt-get update -q -y && apt-get install -y php php-xml php-zip unzip openssh-client composer
+
+RUN usermod -u ${USER_ID:-1000} www-data && groupmod -g ${GROUP_ID:-1000} www-data && mkdir /var/www && chown www-data:www-data /var/www
```
[the remark this answers is not in the document]
Comment (vidarl) [reply Dockerfile]: Okay, abandoned using www-data and /var/www. Dedicated psh user created instead

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (vidarl) [reply Dockerfile]: I remember now why Installed unzip etc... It is needed by composer...
So re-adding git and unzip. Also added pcntl as both `psy/psysh` and `drush/drush` suggest that

**Comment by vidarl** [comment]
This PR looks OK now and can be merged?

**Comment by pjcdawkins** [comment]
🙏 thank you @vidarl

**Pull request merged**